code/_onclick/item_attack.dm

```diff
@@ -45,7 +45,10 @@ avoid code duplication. This includes items that may sometimes act as a standard
 		if(do_surgery(src, user, I))
 			return TRUE
 		else
-			return I.attack(src, user, selected_zone) //This is necessary to make things like health analyzers work. -mattatlas
+			if((I.type in subtypesof(/obj/item/surgery)) || (I.type in list(/obj/item/stack/cable_coil, /obj/item/weldingtool, /obj/item/crowbar, /obj/item/screwdriver, /obj/item/device/multitool))) // It's kinda stupid to whack people with surgery tools on help intent! I assume this is NEVER actually intended.
```

Comment:
Okay, so a few things.
1. We can just do `istype(I, /obj/item/surgery)`. No need for subtypesof fuckery here.
2. There has to be a different way to check for ghetto tools. A hardcoded list like that is NOT sustainable and not an acceptable solution.

Reply:
These are not ghetto tools, these are IPC surgery tools. I'd appreciate tips on how to make this work for just the contents of the mechanical toolbox, but I see no other way this could be implemented for robotics besides a new item flag.

Reply:
I guess the cleanest way is to make a static list of valid tools to hit people with above and then do this:
```
var/static/list/valid_tool_types = list(stuff)
if(... && is_type_in_list(I, valid_tool_types))
```